__tests__/app.test.js

```diff
@@ -410,3 +408,39 @@ describe("GET /api/articles?topic=''", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id (comment_count)", () => {
+  test("200- returns an article object matching a specified id, with a comment count property", () => {
+    return request(app)
+      .get("/api/articles/9")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          title: "They're not exactly dogs, are they?",
+          topic: "mitch",
+          author: "butter_bridge",
+          created_at: "2020-06-06T09:10:00.000Z",
+          votes: 0,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: "2",
+        });
+      });
+  });
+  test("404- responds with NOT FOUND when given valid but non-existent id", () => {
```

Comment:
These two tests (the 404 and 400) already exist above, so you don't need to repeat them